model/proto/apmevent.proto

```diff
@@ -76,15 +74,15 @@ message APMEvent {
   Processor processor = 19;
   HTTP http = 20;
   UserAgent user_agent = 21;
-  Parent parent = 22;
+  string parent = 22;
```

Comment:
I think this should be named `parent_id` not just parent?
```suggestion
  string parent_id = 22;
```

Reply:
Makes sense.
I have also renamed `child` to `childs`.

Reply:
shouldn't it be `children`?

Reply:
It was in the short form, so I didn't extend it. But I can do so.